src/algokit_utils/beta/account_manager.py

```diff
@@ -0,0 +1,200 @@
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from algokit_utils.account import get_dispenser_account, get_kmd_wallet_account, get_localnet_default_account
+from algosdk.account import generate_account
+from algosdk.atomic_transaction_composer import AccountTransactionSigner, TransactionSigner
+from typing_extensions import Self
+
+from .client_manager import ClientManager
+
+
+@dataclass
+class AddressAndSigner:
```

Comment:
I wonder if it's possible to use an [Account](https://github.com/algorandfoundation/algokit-utils-py/blob/28308a0cee6d18917503f13416786223bfb21f4f/src/algokit_utils/models.py#L35) here.

Reply:
The reason I went with `AddressAndSigner` is because `Account` has a private key property, which is not applicable outside of localnet in most cases. Also I want to try to get away from `Account` terminology and instead use `Address` (with account being for HD wallet accounts that are used to derive addresses)